courses/models.py

```diff
@@ -20,6 +20,7 @@ class Program(models.Model):
     live = models.BooleanField(default=False)
     description = models.TextField(blank=True, null=True)
     financial_aid_availability = models.BooleanField(default=False, null=False)
+    ga_tracking_id = models.CharField(max_length=255, blank=True, null=True, default="")
```

Comment:
It seems like we wouldn't want to allow `null` here, is it possible to remove `null=True`, or is it necessary to allow empty strings?